src/Cardano/AddressV3.h

```diff
@@ -24,13 +24,24 @@ class AddressV3 {
     };
 
     enum Kind: uint8_t {
-        Kind_Base = 0,
+        Kind_Base = 0, // spending + staking key
+        //Kind_Base_Script_Key = 1,
```

Comment:
how hard to support those script types address?

Reply:
Test examples are hard to come by, and they are rarely used. If we do not treat them specially, it makes not much sense to add specialized code for them. We do accept different Kinds as valid though, to be on the safe side.